src/features/fireWeather/FireWeatherPage.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { selectAuthenticationReducer } from 'app/rootReducer'
+import {
+  getAuthenticationSuccess,
+  resetAuthentication,
+  getAuthenticationFailed
+} from './slices/authenticationSlice'
+
+export const FireWeatherPage = () => {
+  const { isAuthenticated } = useSelector(selectAuthenticationReducer)
+  const dispatch = useDispatch()
+
+  useEffect(() => {
+    const keycloak = window.Keycloak({
```

Comment:
This is a good solution to start with - we'll need to revisit the moment we have more than one page needing logins, and I suspect we're not dealing correctly with tokens going stale etc. but that's a problem for another day.